source/basic.tex

```diff
@@ -4059,8 +4058,8 @@
 \tcode{void}~(\ref{basic.types}) has three corresponding cv-qualified
 versions of its type: a \defn{const-qualified} version, a
 \defn{volatile-qualified} version, and a
-\defn{const-volatile-qualified} version. The term
-\term{object type}~(\ref{intro.object}) includes the cv-qualifiers
+\defn{const-volatile-qualified} version. An
+object's type~(\ref{intro.object}) includes the cv-qualifiers
```

Comment:
Ah, we have

\newcommand{\grammarterm}[1]{\textit{#1}\xspace}

and the \xspace is probably the culprit (why do we need that in the first place?)